src/pods/edit-realtion/edit-relation.pod.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { DatabaseSchemaVm, RelationVm } from '@/core/providers/canvas-schema';
+import { DropdownOptionVm } from '@/common/components';
+import {
+  mapRelationsTipeToDropdonwVm,
+  mapTablesToDropdonwVm,
+} from './edit-relation.business';
+import { EditRelationComponent } from './edit-relation.component';
+import classes from './edit-relation.pod.module.css';
+
+interface Props {
+  canvasSchema: DatabaseSchemaVm;
+  onChangeRelation: (relation: RelationVm) => void;
+}
+
+export const EditRelationPod: React.FC<Props> = props => {
+  const { canvasSchema, onChangeRelation } = props;
+
+  //TODO: Initialize 'relation' with existing values when the user clicks on 'relation' SVG
+  const [relation, setRelation] = React.useState<RelationVm>({
+    fromFieldId: '',
+    fromTableId: '',
+    toFieldId: '',
+    toTableId: '',
+    type: '1:1',
+  });
+
+  const relationsTipeOptions = mapRelationsTipeToDropdonwVm();
+
+  const tablesNameOptions: DropdownOptionVm[] =
+    mapTablesToDropdonwVm(canvasSchema);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    //TODO: Need validators to avoid passing empty fields
```

Comment:
Cool, formik on it's way :)

Reply:
Let's create a new issue specifically for this TODO.